test/dynamo/test_autograd_function.py

```diff
@@ -699,20 +699,6 @@ def forward(self, x):
     # In the future, we should make the Dynamo test suite actually
     # run on test_autograd.py (it's disabled right now) and delete these.
     def test_smoke_from_test_autograd(self):
-        class Func(torch.autograd.Function):
-            @staticmethod
-            def forward(ctx, x):
-                out0 = x.clone()
-                out1 = x.clone()
-                ctx.mark_non_differentiable(out1)
-                ctx._materialize_non_diff_grads = False
-                return out0, out1
-
-            @staticmethod
-            def backward(ctx, g0, g1):
-                assert g1 is None
```

Comment:
Remove this test since ```assert g1 is None``` should graph break under strict mode, so this is not a proper unit test on capturing backward graph.